Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -569,30 +570,29 @@ ItProjects($select=Name)`;
                         field: "DataLevel", title: "Datatype", width: 150,
                         persistId: "dataLevel",
                         template: dataItem => {
-                            switch (dataItem.DataLevel) {
-                                case "PERSONALDATA":
-                                    return "Persondata";
-                                case "PERSONALDATANDSENSITIVEDATA":
-                                    return "Persondata og følsomme persondata";
-                                default:
-                                    return "Ingen persondata";
-                            }
+                            var contains = [];
+                            _.each(dataItem.SensitiveDataLevels,
+                                dataLevel => {
+                                    switch (dataLevel.SensitivityDataLevel) {
+                                        case "NONE":
+                                            contains.push("Ingen persondata");
+                                            return;
+                                        case "PERSONALDATA":
+                                            contains.push("Almindelige persondata");
+                                            return;
+                                        case "PERSONALDATANDSENSITIVEDATA":
+                                            contains.push("Følsomme persondata");
+                                            return;
+                                        case "PERSONALLEGALDATA":
+                                            contains.push("Straffedomme og lovovertrædelser");
```

Comment:
Mangler Excel template ikke at blive rettet til, eller følger det med af sig selv?

Reply:
Når template ender som en string så kopieres den string til excel output

Reply:
Hvis der skal laves noget er det nok noget med at få filtrering til at virke. Men da det er en nested liste så er det her kendo driller og ikke er til meget hjælp

Reply:
Men er det anderledes end tidligere? Kunne man det før?

Reply:
Der kunne man sortere på hvilken værdi der var valgt. Men der var der også kun én værdi man kunne vælge af gangen og ikke have valgt flere som man kan nu. Det er samme odata problem som der er ved nogle af de andre overblik vi har ændret i hvor vi pludselig har en liste af objekter i stedet for bare en enkelt værdi

Reply:
Man kan godt filtrere i listen (se de nye kommentarer)